cl/import.go

```diff
@@ -270,13 +271,19 @@ func (p *context) initLink(line string, prefix int, f func(inPkgName string) (fu
 			} else {
 				panic(line + ": no specified call convention. eg. //go:linkname Printf C.printf")
 			}
-		} else if c := inPkgName[0]; c >= 'A' && c <= 'Z' {
+		} else if isExported(inPkgName) {
```

Comment:
maybe just remove `if stmt`.

Reply:
Done，I mistakenly thought there was some consideration for non-exported functions here.